projects/plugins/jetpack/extensions/blocks/ai-paragraph/edit.js

```diff
@@ -278,6 +299,31 @@ export default function Edit( { attributes, setAttributes, clientId } ) {
 		}
 	}
 
+	useEffect( ( c1 ) => {
+		if ( ! content ) {
+			return;
+		}
+
+		console.log( 'contentIsLoaded', content );
+
+		// This is to animate text input. I think this will give an idea of a "better" AI.
+		// At this point this is an established pattern.
+		const tokens = content.split( ' ' );
+		for ( let i = 1; i < tokens.length; i++ ) {
+			const output = tokens.slice( 0, i ).join( ' ' );
+			setTimeout( () => updateInnerBlocks( clientId, output ), 50 * i );
+		}
+		setTimeout( () => {
+			updateInnerBlocks( clientId, content );
+		}, 50 * tokens.length );
+	}, [ content ] );
+
+	const TEMPLATE = [
+		[ 'core/paragraph', {
+			placeholder: ' ',
+		} ]
+	];
```

Comment:
placeholder won't be needed once I add a CSS fix to hide the paragraph while it's loading.